tests/test_connector.py

```diff
@@ -1032,6 +1033,9 @@ async def test_tcp_connector_dns_throttle_requests_exception_spread(loop: Any) -
         r1 = loop.create_task(conn._resolve_host("localhost", 8080))
         r2 = loop.create_task(conn._resolve_host("localhost", 8080))
         await asyncio.sleep(0)
+        await asyncio.sleep(0)
```

Comment:
These tests all rely on the event loop overhead introduced by the task. Without the task we need to let it run a few more iterations because the test gets to run sooner now

Reply:
I'd probably just put a delay in (0.01 or something) if it's not guaranteed to be done in the next iteration.

Reply:
8609c35605022650ca07d04fdedd76f708b7b372

I had to leave the two in test_tcp_connector_dns_throttle_requests_cancelled_when_close since it failed otherwise

Reply:
I ended up reverting them since the test was only reliable as when it waited X iterations of the event loop instead of time